packages/js/product-editor/assets/icons/section_images.svg

```diff
@@ -0,0 +1,20 @@
+<svg
+	width="24"
```

Comment:
```suggestion
```

Since these `svg` files are going to be used as images I thing it's better to not fix their sizes here and let the `<img src="https://.../section_images.svg" height="32" width="32" />` controls the sizes.